### PR TITLE
Updated the readme file

### PR DESCRIPTION
**Pull request by Kentoseth**
I added specific instructions for launching a working/rendering version of the website using the Python3 HTTP server

Signed-off by: Mohamed Qas <kentoseth@devcroo.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,21 @@ linuxcontainers.org website.
 ## Generating the website
 
     ./generate
+    
+## Launching website within container
+
+Make sure that the container has access to the  website folder that you just cloned.
+
+You can use a bind-mount for the folder if necessary (host => container OR container => host)
```
Comment (techtonik) [inline README.md]: Can you write down this optional command here?

File: README.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ linuxcontainers.org website.
...
+
+After generating the website(above), run these commands(Ubuntu-specific) within the container:
+
+    cd output
+    python3 -m http.server 8777
+    
+You now need to obtain the IP address for the specific container. You can now navigate to the site (from the host) with the following example IP address:
+
+> 185.5.3.12:8777
```
Comment (techtonik) [inline README.md]: It is global IP, isn't it? Why it will port to the local guest?

**Comment by Kentoseth** [comment]
Hey @techtonik . The IP address is the container IP address. The port is specified in the Python3 HTTP server.

The reason why you use the container IP address is because when running the Python3 command, the server renders on 0.0.0.0 and not localhost.

Does that answer your question?

**Comment by techtonik** [comment]
I understand that Python will listen on all interfaces (0.0.0.0) on port 8777, so if one interface has an external IP 185.5.3.12 reachable from some network, then the web server will be available at 185.5.3.12:8777 at that network. However, it is not clear how to figure out this IP and how to bind-mount folder.

**Comment by Kentoseth** [comment]
Here is an example of a bind-mount from container to host (meaning, you can access the container-folder from the host-system):

`sudo mount --bind /var/lib/lxc/container1/rootfs/home/ubuntu/ /path/to/folder`

And here is the command to show information about the container:

`sudo lxc-info --name container1`

**Comment by techtonik** [comment]
Nice. Can I use bind-mount to forward socket or port from container to host?

**Pull request merged**